Documentation/disaster-recovery.md

```diff
@@ -50,7 +50,26 @@ plane.
 The `bootkube recover` subcommand does not recover a cluster directly. Instead,
 it extracts the control plane configuration from an available source and
 renders manifests in a format that `bootkube start` can use invoked to reboot
-the cluster.
+the cluster. There are two available sources to choose from: etcd or an API
+server.
+
+### What does bootkube recover do?
+
+Bootkube attempts to read the configuration from an existing backend etcd or API
+server. On success, the tool renders a modified bootstrap control plane to
```

Comment:
what does `modified` mean? also can we clarify that the tool renders manifests (basically change control plane to control plane manifests).